File: internal/content/testdata/bench/typescript/count-vowels-in-string.ts
function count_vowels(s: string): number {
    const vowels = new Set(['a','e','i','o','u','A','E','I','O','U']);
    let count = 0;
    for (const ch of s) {
        if (vowels.has(ch)) count++;
    }
    return count;
}

File: internal/content/testdata/bench/go/count-vowels-in-string.go
func count_vowels(s string) int {
	vowels := map[rune]bool{
		'a': true, 'e': true, 'i': true, 'o': true, 'u': true,
		'A': true, 'E': true, 'I': true, 'O': true, 'U': true,
	}
	count := 0
	for _, ch := range s {
		if vowels[ch] {
			count++
		}
	}
	return count
}
